systemd/install-relay

```diff
@@ -359,29 +208,21 @@ installServices() {
 	echo "INSTALL SERVICES: DONE"
 }
 
-chainVariant="mainnet"
-case "$1" in
-	ethereum)
-		_gas=150000
-		;;
-	optimism)
-		_gas=250000
-		;;
-	arbitrum)
-		_gas=5000000
-		;;
-	help | --help | -h)
-		usage
-		;;
-	*)
-		echo >&2 "Chain type must be one of [ethereum|optimism|arbitrum]"
-		usage
-		;;
-esac
-chainType="$1"
-configOverrides+=(".ethereum.type = \"$1\"")
-configUpdates+=(".options.ethGas = ${_gas}")
-shift
+if [[ "$1" =~ ^ethereum|optimism|arbitrum$ ]]; then
```

Comment:
I think if you're wanting to enforce strictness on `$1` here, you need parens

```
if [[ "$`" =~ ^(ethereum|optimism|arbitrum)$ ]]; then
  ....
fi 
```
e.g. in the above if you pass `sandwich-optimimsm-between-other-words` this logic gets triggered